### PR TITLE
adjusted rest examples to new sdk version

### PR DESCRIPTION
**Pull request by patrickwieth**
As discussed here:

https://github.com/cosmos/sdk-application-tutorial/issues/80

this commit might bring back happiness to the people (using the rest examples)

### CODE REVIEW

File: tutorial/run-rest.md [hunk 1]
```diff
@@ -24,18 +24,29 @@ $ curl -s http://localhost:1317/auth/accounts/$(nscli keys show jack -a)
...
-$ curl -XPOST -s http://localhost:1317/nameservice/names --data-binary '{"base_req":{"from":"jack","password":"foobarbaz","chain_id":"namechain","sequence":"2","account_number":"0"},"name":"jack1.id","amount":"5nametoken","buyer":"cosmos127qa40nmq56hu27ae263zvfk3ey0tkapwk0gq6"}'
-# > {"check_tx":{"gasWanted":"200000","gasUsed":"1242"},"deliver_tx":{"log":"Msg 0: ","gasWanted":"200000","gasUsed":"2986","tags":[{"key":"YWN0aW9u","value":"YnV5X25hbWU="}]},"hash":"098996CD7ED4323561AC9011DEA24C70C8FAED2A4A10BC8DE2CE35C1977C3B7A","height":"23"}
+curl -XPOST -s http://localhost:1317/nameservice/names --data-binary '{"base_req":{"from":"'$(nscli keys show jack -a)'","chain_id":"namechain"},"name":"jack1.id","amount":"5nametoken","buyer":"'$(nscli keys show jack -a)'"}' > unsignedTx.json
+
+# Then sign this transaction
+# NOTE: In a real environment the raw transaction should be signed on the client side. Also the sequence needs to be adjusted, depending on what the query of alice's account has shown.
+nscli tx sign unsignedTx.json --from jack --offline --chain-id namechain --sequence 1 --account-number 0 > signedTx.json
```
Comment (yangyanqing) [inline tutorial/run-rest.md]: `sequence` and `account-number` should be same with result of `curl -s http://localhost:1317/auth/accounts/$(nscli keys show jack -a)`

File: tutorial/run-rest.md [hunk 2]
```diff
@@ -24,18 +24,29 @@ $ curl -s http://localhost:1317/auth/accounts/$(nscli keys show jack -a)
...
 
 # Set the data for that name that jack just bought
 # NOTE: Be sure to specialize this request for your specific environment, also the "owner" and "from" should be the same address
-$ curl -XPUT -s http://localhost:1317/nameservice/names --data-binary '{"base_req":{"from":"jack","password":"foobarbaz","chain_id":"namechain","sequence":"3","account_number":"0"},"name":"jack1.id","value":"8.8.4.4","owner":"cosmos127qa40nmq56hu27ae263zvfk3ey0tkapwk0gq6"}'
+$ curl -XPUT -s http://localhost:1317/nameservice/names --data-binary '{"base_req":{"from":"'$(nscli keys show jack -a)'","chain_id":"namechain"},"name":"jack1.id","value":"8.8.4.4","owner":"'$(nscli keys show jack -a)'"}' > unsignedTx.json
 # > {"check_tx":{"gasWanted":"200000","gasUsed":"1242"},"deliver_tx":{"log":"Msg 0: ","gasWanted":"200000","gasUsed":"1352","tags":[{"key":"YWN0aW9u","value":"c2V0X25hbWU="}]},"hash":"B4DF0105D57380D60524664A2E818428321A0DCA1B6B2F091FB3BEC54D68FAD7","height":"26"}
 
+# Again we need to sign and broadcast
+nscli tx sign unsignedTx.json --from jack --offline --chain-id namechain --sequence 2 --account-number 0 > signedTx.json
```
Comment (yangyanqing) [inline tutorial/run-rest.md]: Same with upon.

**Comment by michaelfig** [comment]
Is the only thing blocking this PR the comments from @yangyanqing ?

This was a problem for me, notably because I just cloned the tutorial, built it, and tried to run the REST instructions.  Is there anything I can do to help this move forward?

**Comment by jackzampolin** [comment]
@patrickwieth sorry for not moving on this sooner. We are working on a full overhaul of the tutorial right now.

**Pull request merged**